client/src/components/UsedOnTable/UsedOnTable.js

```diff
@@ -4,6 +4,13 @@ import classnames from 'classnames';
 import i18n from 'i18n';
 import provideUsedOnData from './provideUsedOnData';
 
+import { useTabFirstShow } from 'hooks/useTabContext';
```

Comment:
All the changes in the file are here for demo purposed. We'll need to revert them once we are done.